frontend/src/contexts/shopping-cart-context/shoppingCartContext.tsx

```diff
@@ -96,7 +100,7 @@ export const ShoppingCartProvider = ({ children }: shoppingCartChildren) => {
         setPaymentMethod('');
         setUserOptionsOpen(null);
         handleMobileMenuClose();
-        navigateUserSignIn();
+        navigate('/user/signin');
```

Comment:
use the constant value instead 🙊

Reply:
I was gonna to do it when i refactor context.

Reply:
ok 🙈